packages/plugin-server-session/session.js

```diff
@@ -48,8 +31,6 @@ module.exports = {
 const sendSessionSummary = client => sessionCounts => {
   const releaseStage = inferReleaseStage(client)
 
-  if (!client.config.autoCaptureSessions) return
```

Comment:
This was the nub of the issue! At this point we don't know if the sessions we are about to report are manually or automatically captured.